scirius/settings.py

```diff
@@ -161,6 +161,7 @@
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
How do you 'start the project" ? On the code side, your patch looks ok.

Reply:
I use a Docker image based off of python:2.7-alpine3.8. The command (in a shell script) is `gunicorn -w "$(($(nproc --all)*2+1))" -t 120 -b 0.0.0.0:8000 --preload scirius.wsgi`.

Reply:
ok, yes collectstatic will need that variable for sure.

Reply:
I've built my own Dockerfile, but only because I couldn't get https://github.com/StamusNetworks/scirius-docker to work; it has collectstatic as one of its commands and errors in the same place.